addons/pos_razorpay/static/src/app/payment_razorpay.js

```diff
@@ -161,7 +162,9 @@ export class PaymentRazorpay extends PaymentInterface {
                 paymentLine.razorpay_reference_no = response?.externalRefNumber;
                 paymentLine.razorpay_reverse_ref_no = response?.reverseReferenceNumber;
                 paymentLine.transactionId = response?.txnId;
-                paymentLine.payment_date = response?.createdTime;
+                paymentLine.payment_date = new DateTime(response?.createdTime).toFormat(
```

Comment:
I do not know what type is `createdTime` but to create a DateTime from something, look at this class https://github.com/odoo/odoo/blob/82aab0aefb45045264ab5fb2238be99f18a84190/addons/web/static/lib/luxon/luxon.js#L5739

There are method for a lot of different method for different input